coforma/swift-tech-challenge
# Confirmation Page Styling

# Title
Confirmation Page Styling
# Summary

This adds some styling to the confirmation page. Will do a further refining later with design

## Type of change

Please delete options that are not relevant.


# How To Test

- Click a college to apply to
- append /confirmation to the url
- See the new changes!

# Checklist:
